### PR TITLE
Incomplete todos counter

### PR DESCRIPTION
**Pull request by JamieZafar**

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -50,13 +51,20 @@ function makeButton(content, cssClass, onClick) {
     return button;
 }
 
+function countIncompletes(todos, countLabel) {
+    countLabel.textContent = todos.filter(function(todo) {
```
Comment (lclarksl) [inline public/main.js]: Nice use of higher order functions here! I think in this case, `reduce` would be more appropriate.

File: public/main.js [hunk 2]
```diff
@@ -50,13 +51,24 @@ function makeButton(content, cssClass, onClick) {
     return button;
 }
 
+function countIncompletes(todos, countLabel) {
+    countLabel.textContent = todos.reduce(function(p, c) {
+        if (!c.isComplete) {
```
Comment (lclarksl) [inline public/main.js]: Nice! I think a ternary operator would be neater here.

**Comment by lclarksl** [comment]
:shipit:

**Comment by cuthullu** [comment]
Very tidy :+1: :shipit:

**Comment by JamieZafar** [comment]
cheers :100:

**Pull request merged**